xena/pages/login_page.py

```diff
@@ -49,5 +49,5 @@ def dev_auth(self, uid=None):
             uid = util.get_admin_uid()
         app.logger.info(f'Logging in to El Diablo as UID {uid}')
         self.wait_for_element_and_type(LoginPage.USERNAME_INPUT, uid)
-        self.wait_for_element_and_type(LoginPage.PASSWORD_INPUT, app.config['DEVELOPER_AUTH_PASSWORD'])
+        self.wait_for_element_and_type(LoginPage.PASSWORD_INPUT, app.config['DEV_AUTH_PASSWORD'])
```

Comment:
@pfarestveit @paulkerschen - You'll need to update dev_auth keys in your local config.

Reply:
+1, thanks for the note.